app/web/controllers/audio_files.py

```diff
@@ -28,28 +62,34 @@ def __handle_prefix(self, prefix: str) -> tuple[list[str], Path]:
     @cherrypy.expose
     @cherrypy.tools.allow(methods=['GET'])
     @cherrypy.tools.authenticate()
-    @cherrypy.tools.authorize(role=UserRole.MODERATOR)
-    def index(self, prefix: str = ''):
+    @cherrypy.tools.authorize(role=UserRole.RUNNER)
+    def index(self, prefix: str = '', file_callback: Optional[str] = None):
```

Comment:
Should the role be also updated for `upload_files` and `create_dir`?

Reply:
For now I think no